tests/specs/mtx/downlinkCommands.spec.js

```diff
@@ -5,6 +5,25 @@ import fixture from '../../fixtures/main.js';
 import {downlinkCommands} from '../../fixtures/mtx/commands.js';
 
 
+test.describe('mtx downlink commands - parse hex dumps', () => {
+    test.beforeEach(async ( {page, baseURL} ) => {
+        await page.goto(baseURL);
+        await new MainPage(page).selectCodec(fixture.codecType.options.MTX);
+    });
+
+    test.afterEach(async ({page}) => page.getByLabel(fixture.logs.buttons.deleteLogs).click());
```

Comment:
```suggestion
    test.afterEach(async ( {page} ) => page.getByLabel(fixture.logs.buttons.deleteLogs).click());
```

also how about either add await or remove async?

Reply:
removed and in other places too